frontend/src/views/camp/Admin.vue

```diff
@@ -22,7 +22,7 @@ Admin screen of a camp: Displays details & periods of a single camp and allows t
         </v-col>
         <v-col cols="12" lg="6">
           <camp-categories :camp="camp" :disabled="!isManager" />
-
+          <camp-activity-progress-labels :camp="camp" :disabled="!isManager" />
```

Comment:
In the API we allow members to edit the labels. The same is true for the categories and possibly more. Should be resolved in #3440.